cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/sourcecontrol/ApplicationManager.java

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
please make sure all checkstyle errors are fixed (there a several in the unit tests too)

Reply:
Fixed